registers.md

```diff
@@ -329,61 +334,61 @@ if TYPE == 2:
 * CMD = 1: write Register
 * CMD = 2: read Register
 * CMD = 3: chain Inactive
-### Frame Lenght
-Total Frame Lenght excluding preamble.
+### Frame Length
+Total Frame Length excluding preamble.
 ### Data
-Depend on TYPE/CMD, see detailed frames below.
+Depends on TYPE/CMD, see detailed frames below.
 ### CRC
-Can be CRC5 or CRC16 depending on TYPE/CMD, see detailled frames below.
+Can be CRC5 or CRC16 depending on TYPE/CMD, see detailed frames below.
 ## Response
 ![](images/response.svg)
 
-All Responses have fixed lenght : 9 bytes.
+All Responses have fixed length : 9 bytes.
 ### Preamble
-All command have a fixed 2 bytes preamble: 0xAA 0x55
+All commands have a fixed 2 byte preamble: 0xAA 0x55
 ### TYPE
 * TYPE = 0: respond to a command
 * TYPE = 4: respond to a job (nonce)
 ### Data
 Depends on TYPE, see detailed frames below.
 ### CRC5
-CRC 5 bits with polynomial 0x05, intial value 0x1F, no reflection, no final XOR of the full Frame excluding preamble.
+CRC 5 bits with polynomial 0x05, initial value 0x1F, no reflection, no final XOR of the full Frame excluding preamble.
 ## Set Chip Address
-On reset all chip have a logical address of 0. In order to access to a specific chip later, we must give them different address.
+On reset, all chips have a logical address of 0. In order to access to a specific chip later, we must give them different addresses.
 
-Warning : Chip Address are different to Chip index on the chain. IT is a logical concept configurable by software.
+**Warning**: Chip *addresses* are different from the chip *index* on the chain. It is a logical concept configurable by software.
 
-To set Chip Address of all chip one by one, we must not send command to ALL chip, just to the chip with Address = 0, so the first chip on the chain will get the command and not propagate it downward.
+To set Chip Address of all chips one by one, we **must not** send commands to ALL chips, just to the chip with Address = 0, so the first chip on the chain will get the command and not propagate it downward.
 
 The Set Chip Address Command format is:
 
 ![](images/set_chip_address.svg)
 
-No Response is replied by the chip.
+No Response is returned by the chip.
 ## Write Register
 The Write Register Command format is:
 
 ![](images/write_register.svg)
 
-No Response is replied by the chip.

```

Comment:
here also, keep "Response" with upper case R, as others. Because "Response" is a chapter described above.